test/models/work_test.rb

```diff
@@ -0,0 +1,52 @@
+require "test_helper"
+
+describe Work do
+
+  it 'can be instantiated' do
+    (Work.new).must_be_kind_of Work
```

Comment:
This should test creating a work and that it's valid.

```suggestion
    work = works(:test_book)
    expect(work).must_be_kind_of Work
    expect(work.valid?).must_equal true
```